mac viewer/closer and conditional statement

jstitlow opened the pull request:
Added lines to open and close the images with Preview on Macs. 
Also added a conditional statement to img_fpaths which omits files that have already been scored. This modality allows you to resume working on a set of images if the app is closed for some reason.

carandraug commented:
If you have more than one preview window, which windows get closed when you close them like that?

I have mixed fellings about the filtering of images already scored. I envisioned this to have the Makefile handle that, and possibly even check if the mages on figure had changed. But I guess no one is using it in that manner.

jstitlow commented:
If you have more than one preview window, which windows get closed when you
close them like that?
   > All of them, the command closes Preview. But the app can only open one
(unique) image at a time, so it would only close background images open in
Preview.
I envisioned this to have the Makefile handle that, and possibly even check
if the mages on figure had changed. But I guess no one is using it in that
manner.
   > The Makefile approach is great but if the app quits it is cumbersome
to remake the directory structure. Filtering the scored images is a simple
way to resume the analysis.

On Wed, Mar 4, 2020 at 9:57 AM Carnë Draug <notifications@github.com> wrote:

> If you have more than one preview window, which windows get closed when
> you close them like that?
>
> I have mixed fellings about the filtering of images already scored. I
> envisioned this to have the Makefile handle that, and possibly even check
> if the mages on figure had changed. But I guess no one is using it in that
> manner.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/carandraug/mRNA-localisation-screening/pull/5?email_source=notifications&email_token=ABMMCBRW5FSSDN7HRDFQRPTRFZT5RA5CNFSM4LBE52W2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENYIKAY#issuecomment-594576643>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMMCBXJU3CZUN4SS6M4NFLRFZT5RANCNFSM4LBE52WQ>
> .
>

jstitlow commented:
Scroll bar is cool!

I merged and added code to open the next image by default when save and exit button is clicked. Opening the next image is inevitable, no reason to have to click the open button. 

Also switched the open image code to a PIL command as a potential platform independent solution.
